src/Ui.GraphiQL/GraphiQLOptions.cs

```diff
@@ -33,16 +33,16 @@ public class GraphiQLOptions
 
     /// <summary>
     /// Enables the header editor when <see langword="true"/>.
-    /// Not supported when <see cref="ExplorerExtensionEnabled"/> is <see langword="true"/>.
     /// </summary>
     /// <remarks>
     /// Original setting from <see href="https://github.com/graphql/graphiql/blob/08250feb6ee8335c3b1ca83a912911ae92a75722/packages/graphiql/src/components/GraphiQL.tsx#L186">GraphiQL</see>.
     /// </remarks>
     public bool HeaderEditorEnabled { get; set; } = true;
 
     /// <summary>
-    /// Enables the explorer extension when <see langword="true"/>.
+    /// This property has no effect.
     /// </summary>
+    [Obsolete("This property has no effect and will be removed in a future version.")]
     public bool ExplorerExtensionEnabled { get; set; } = true;
```

Comment:
There was no code tied to this property within the entire codebase.  So it could not be disabled.